docs/topics/modules/using_stack_report.adoc

```diff
@@ -0,0 +1,62 @@
+[#using_stack_report]
+= Using your Stack Report
+
+{osio} Analytics helps you make informed choices with respect to your development stack and its component libraries by analyzing your stack at the development stage, within your IDE, and at the deployment stage.
+
+== Development Stage
+{osio} Analytics analyzes your stack at the development stage and flags components with security vulnerabilities in your Che workspace. It also recommends a suitable, secure, alternate version for your stack.
+
+.Prerequisite
+
+Ensure that you use  link:e2e_workshop.html#create_che_workspace-hello-world[Che workspace] to develop an application.
+
+.Procedure
+
+You can access inputs from {osio} Analytics within the Che IDE as follows:
+
+. In your Che workspace, open the manifest file of your project, for example, `pom.xml` for a Maven Stack, `package.json` for NPM, or `requirements.txt` for Python. {osio} analyzes the stack, flags (image:che_cve_issue.png[title="CVE Flag"]) components with security vulnerabilities, and recommends an alternative secure version.
+. Mouse hover the flagged component to see the CVE for the flagged component and the recommended alternative secure version.
++
+image::che_alt_recommendations.png[Stack Recommendations on Che]
++
+. Update the component to the recommended version.
+
+== Deployment Stage
+At the Deployment stage, {osio} Analytics provides a detailed Stack Analysis Report. It analyzes various parameters of your entire stack and recommends alternative and additional components suitable for your stack.
+
+.Prerequisites
+
+Ensure that you deploy your project and <<creating_a_new_pipeline, create a build pipeline>> for your project.
+
+.Procedure
+
+* In the *Stack Report Recommendations* section of your space dashboard, click *Show complete stack report* to access the detailed stack report for your manifest files.
```

Comment:
The capitalization problem is in the app, too.

Reply:
Yes, there are a lot of inconsistencies wrt cases, US>UK english. I have pointed them out. Also, UX is changing, so a lot of these labels may just disappear. I have however changed references to stack reports to all small in the doc, except when it  refers to a field in UX or is part of a Title/Heading.